doc/lua_api.txt

```diff
@@ -835,8 +821,8 @@ the global `minetest.registered_*` tables.
 
 * `minetest.unregister_item(name)`
     * Unregisters the item name from engine, and deletes the entry with key
-    * `name` from `minetest.registered_items` and from the associated item
-    * table according to its nature: `minetest.registered_nodes[]` etc
+      `name` from `minetest.registered_items` and from the associated item
```

Comment:
Making this a nested list would be nicer; i.e.

* Unregisters the item ...
  * `name` from ...

Reply:
Good point

Reply:
I agree with SmallJoker, this is fine.